server/odc-server/src/main/java/com/oceanbase/odc/server/web/controller/v2/ProjectController.java

```diff
@@ -82,6 +82,12 @@ public PaginatedResponse<Project> listProjects(
         return Responses.paginated(projectService.list(params, pageable));
     }
 
+    @ApiOperation(value = "listProjects", notes = "List all projects")
+    @RequestMapping(value = "/projects/all", method = RequestMethod.GET)
+    public ListResponse<Project> listAllProjects(@RequestParam(required = false, name = "archived") Boolean archived) {
```

Comment:
Why do we need a new API? Is it possible to reuse the list projects API?

Reply:
I find that the exists `/projects` api always return projects that current user granted to, and will always return related infos such as member\env etc. But in this scenes, I just need all projects with only basic info (ID  and name). 
One way is as you say, reuse this api, and refact the params, but the front-end should change their codes.  The other way may easier, a new api.
So, which way is better?